src/server/protocol.ts

```diff
@@ -1621,6 +1633,11 @@ namespace ts.server.protocol {
          * List of last known projects
          */
         knownProjects: ProjectVersionInfo[];
+        /**
+         * If true, response specifies whether or not each file in each project
+         * is the result of a project reference redirect

```

Comment:
Shouldn't it be `source of project reference redirect`?

Reply:
Fixed.